### DISCUSSION TITLE
Using Banks without buttons (switching banks via code)

### DISCUSSION THREAD

**Discussion by ransa808** [tttapa/Control-Surface#376] [General]
Hi
I want to implement bankable CCpots, but I don't want to use any physical buttons, I want to switch the banks via certain conditions in the code.. it is possible ?

**Comment by tttapa** [comment]
Yes, you can use the [`Bank::select()`](https://tttapa.github.io/Control-Surface-doc/Doxygen/db/dbd/classBank.html#a801ab5dedea8200f1cdb26533cb6f78b) method. `bank.select(0)` selects the first bank, `bank.select(1)` the second, etc.

**Reply by ransa808** [reply]
Thank you ! will check it out
